packages/material-ui-lab/src/TimelineContent/TimelineContent.test.js

```diff
@@ -1,21 +1,31 @@
 import * as React from 'react';
-import { getClasses, createMount, describeConformance } from 'test/utils';
+import { expect } from 'chai';
+import { createClientRender, createMount, describeConformanceV5 } from 'test/utils';
 import Typography from '@material-ui/core/Typography';
-import TimelineContent from './TimelineContent';
+import Timeline from '../Timeline';
+import TimelineContent, { timelineContentClasses as classes } from './index';
```

Comment:
Best to test the public API

```suggestion
import Timeline from '@material-ui/lab/Timeline';
import TimelineContent, { timelineContentClasses as classes } from '@material-ui/lab/TimelineContent';
```